packages/calypso-e2e/src/element-helper.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+
+/**
+ * Type dependencies
+ */
+import { Page } from 'playwright';
+
+/**
+ * Waits for the element specified by the selector to become enabled.
+ *
+ * There are two definitions of disabled on wp-calypso:
+ * 		- traditional: set the `disabled` property on element.
+ * 		- aria: using the `aria-disabled` attribute.
+ *
+ * Playwright's `isEnabled` method does not look into `aria-disabled = "true"` when determining
+ * actionability. This means test steps may fail if it depends on the built-in `isEnabled` to
+ * assert actionability.
+ *
+ * This function will check for both the`disabled` property and `aria-disabled="false"` attributes
+ * to determine whether the element is enabled.
+ *
+ * @param {Page} page Page object.
+ * @param {string} selector Selector of the target element.
+ * @param {{[key: string]: number}} options Object parameter.
+ * @param {number} options.timeout Timeout to override the default value.
+ *
+ */
+export async function waitForElementEnabled(
+	page: Page,
+	selector: string,
+	options?: { timeout?: number }
+): Promise< void > {
+	const elementHandle = await page.waitForSelector( selector, options );
+	await Promise.all( [
+		elementHandle.waitForElementState( 'enabled', options ),
+		page.evaluate( ( element: any ) => element.ariaDisabled !== 'true', elementHandle ),
```

Comment:
Almost there! 😄 We still need to use the `wait` method as it should poll the condition fn, and then either resolve or throw an error if the condition is not met.  With `page` available we can finally do:
```suggestion
		page.waitForFunction( ( element: any ) => element.ariaDisabled !== 'true', elementHandle ),
```

Reply:
Right yeah, that escaped me 🤕 Fixed with latest push!